bbq/opcode/instructions.yml

```diff
@@ -937,3 +937,50 @@
     push:
       - name: "string"
         type: "string"
+
+# Attachment instructions
+
+- name: "getTypeKey"
```

Comment:
Reg. naming: `getTypeIndex` might better complement the other existing instruction `getIndex` (given both are index accesses, with the syntax `A[B]`). Same for the set/remove instructions below.